src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php

```diff
@@ -34,12 +36,12 @@ function let(ObjectRepository $countryRepository, FormFactoryInterface $factory)
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('Sylius\Bundle\AddressingBundle\Form\EventListener\BuildAddressFormSubscriber');
+        $this->shouldHaveType(BuildAddressFormSubscriber::class);
```

Comment:
@michalmarcinkowski `MyClass::class` will always refer to `Current\Namespace\MyClass` unless you add a `use` statement.

Anyway, it also helps you to see whether the class (or interface) really exists as IDE also highlights unexisting classes.